autoptsclient_common.py

```diff
@@ -503,6 +504,17 @@ def get_test_case_description(cache, test_case_name):
             return test_case.find('description').text
 
 
+def get_result_color(status):
```

Comment:
Add more coloring for different status like PTS TIMEOUT, BTP TIMEOUT and BTP ERROR?

Reply:
I think that might be too much, with coloring for PASS, FAIL, INCONC and no coloring for other statuses it's clearer, otherwise, it looks too rainbowish.